rpa-home/modules/ROOT/pages/invoke-rpa-from-flow-build.adoc

```diff
@@ -0,0 +1,47 @@
+
+# Build an Automation
+
+Implement your automation in RPA Builder.
+
+## Before You Begin
+
+* The following variable types of RPA Builder Activity Parameters are supported as input or output variables of an Invocable Action by Salesforce:
++
+[cols="1,1,2"]
+|===
+|*Activity Parameter Type*
+|*Input/Output Variable Type*
+|*Additional Information*
+
+|Alphanumeric
+|ID, Long, String
+|You can use comma-seperated strings within strings to replace an array. In this case, name the variable in plural and add the suffix CommaSeperatedList to the variable names; for examplespecialAccountsCommaSeperatedList.
```

Comment:
```suggestion
|You can use comma-seperated strings within strings to replace an array. In this case, name the variable in plural and add the suffix `CommaSeperatedList` to the variable names; for example, `specialAccountsCommaSeperatedList`.
```

Reply:
In the above text, "Seperated" is spelled wrong. Hopefully, this isn't in the product. If so, it should be corrected.